databases/week2/DB-W2.sql

```diff
@@ -0,0 +1,44 @@
+use HYF_W2_part3;
+SELECT * FROM `user`;
+SELECT * FROM `status`;
+SELECT * FROM `task`;
+SELECT * FROM `user_task`;
+
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT `title` AS 'list of the tasks assigned to the user'
+FROM `task`
+INNER JOIN `user_task`
+ON `task`.`id` = `user_task`.`task_id`
+INNER JOIN `user`
+ON `user`.id = `user_task`.`user_id`
+WHERE `user`.`email` LIKE '%@spotify.com%'
+group by `task`.`title`;
+
+-- Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT `title` AS 'list of the not-started tasks assigned to Donald Duck'
+FROM `task`
+INNER JOIN `user_task`
+ON `task`.`id` = `user_task`.`task_id`
+INNER JOIN `user`
+ON `user`.id = `user_task`.`user_id`
+INNER JOIN `status`
+ON `task`.`status_id` = `status`.`id`
+WHERE `user`.`name` = 'Donald Duck' 
+AND `status`.`name` = 'Not started';
+
+-- Get all the tasks for 'Maryrose Meadows' that were created in september 
+SELECT `title` AS 'tasks for Maryrose Meadows that were created in september'
+FROM `task`
+INNER JOIN `user_task`
+ON `task`.`id` = `user_task`.`task_id`
+INNER JOIN `user`
+ON `user_task`.`user_id` = `user`.`id`
+WHERE `user`.`name` = 'Maryrose Meadows'
+AND month(`task`.`created`) = 9;
```

Comment:
add some infomation atleast user name, status name .
```
SELECT task.id, task.title AS 'Task Name', user.name AS 'user', status.name AS status
FROM user_task
JOIN task ON user_task.task_id = task.id
JOIN user ON user_task.user_id = user.id
JOIN status ON task.status_id = status.id
WHERE status.name = "Not started" AND user.name = 'Donald Duck';
&
```
SELECT task.id,user.name, task.title, task.created
FROM task
JOIN user_task ON task.id = user_task.task_id
JOIN user  ON user.id = user_task.user_id
WHERE user.name LIKE 'Maryrose Meadows' AND MONTH(created)=09;
```
```